lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -636,10 +648,11 @@ export abstract class CacheManager implements ICacheManager {
      * @param environment
      * @param authScheme
      */
-    readCacheRecord(account: AccountInfo, clientId: string, scopes: ScopeSet, environment: string, authScheme: AuthenticationScheme, keyId?: string): CacheRecord {
+    readCacheRecord(account: AccountInfo, clientId: string, request: BaseAuthRequest, environment: string): CacheRecord {
```

Comment:
This is public surface and may be a breaking change

Reply:
I'm fine with breaking changes in common in the name of cleaner code, we just need to make the changefiles aware if that's the direction we're taking.